test_runner/regress/test_basebackup.py

```diff
@@ -26,22 +26,37 @@ def test_basebackup_cache(neon_env_builder: NeonEnvBuilder):
     ps = env.pageserver
     ps_http = ps.http_client()
 
+    storcon_managed_timelines = (env.storage_controller_config or {}).get(
```

Comment:
maybe add two variants of the test, one with `timelines_onto_safekeepers` enabled, one with it disabled?

Reply:
I don't think it's worth it because it will be basically the same test with very little difference. It's more like the test is fragile to some code changes, the cache itself works fine. I think we can just delete this check when we move to `--timelines-onto-safekeepers` completely

I'll think about how to make the test more robust to unrelated code changes